py-polars/polars/internals/dataframe/frame.py

```diff
@@ -6805,6 +6805,57 @@ def iterrows(
             for i in range(self.height):
                 yield self.row(i)
 
+    def iterslices(self, n_rows: int = 10_000) -> Iterator[DataFrame]:
```

Comment:
Can we name this `iter_slices`?

Reply:
Sure, though that doesn't really match-up with `iterrows`; should we change both for `0.16`? I agree it looks cleaner to have a little clarifying visual space between the `iter` and _what_ is being iterated over ;)

* `iter_rows`
* `iter_slices`

Reply:
Yes, let's do that with `iter_rows` as well.

I hope to release the last `0.15.x` today and then we can merge the queue.